apps/daimo-mobile/src/view/screen/receive/CreateRequestScreen.tsx

```diff
@@ -19,7 +19,7 @@ export default function CreateRequestScreen() {
   const [account] = useAccount();
   if (account == null) return null;
 
-  const url = formatDaimoLink({ type: "account", addr: account.address });
+  const url = formatDaimoLink({ type: "account", account: account.name });
```

Comment:
looks good

we have to be very careful to maintain the invariant that `account.name` matches what's in NameManager to ensure payment goes to the right place

right now that invariant is good, just have to take care in case we ever allow name-reuse or name change